tsconfig-base.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "target": "es5",
+    "lib": ["dom", "es6", "dom.iterable", "scripthost"], // es6 stuff is polyfilled by stimulus
+    "sourceMap": true,
+    "moduleResolution": "node",
+    "skipLibCheck": true
```

Comment:
There are probably a few more flags you wanna include here, eg `isolatedModules`